packages/drivers/file-driver/src/fileDocumentStorageService.ts

```diff
@@ -292,9 +284,9 @@ export function FileSnapshotWriterClassFactory<TBase extends ReaderConstructor>(
             }
 
             for (const blobName of Object.keys(snapshotTree.blobs)) {
-                const contents = await this.read(snapshotTree.blobs[blobName]);
+                const contents = bufferToBase64(await this.readBlob(snapshotTree.blobs[blobName]));
```

Comment:
bufferToBase64 is wrong - the old code was decoding base64 result of read(). New code calls readBlob() that already did decoding. So all you need here is to convert buffer (assumes it contains utf-8 data) to string, but no base64 manipulations at all.
I.e. IsoBuffer.from(input, "utf-8").toString()

Reply:
is using bufferToString() ok?